src/exception.cr

```diff
@@ -21,6 +21,11 @@ class Exception
   property callstack : CallStack?
 
   def initialize(@message : String? = nil, @cause : Exception? = nil)
+    {% if flag?(:release) %}
```

Comment:
Ooops, this will get re-initialized on every `Exception.new`.

This might be moar up your alley:
```cr
class_getter colorize : Bool = {% if flag?(:release) %} STDERR.tty? {% else %} false {% end %}
```

Reply:
Ah, that's neat. Will change now!